emsdk.py

```diff
@@ -2584,10 +2591,9 @@ def construct_env(tools_to_activate):
         else:
           assert False
 
-      errlog(key + ' = ' + value)
+    errlog('')
   return env_string
 
-
 def error_on_missing_tool(name):
```

Comment:
Doesn't flake8 complain about this if you don't have two lines between each function?

Reply:
I force pushed the formatting fixes from flake